e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    let config;
+
+    beforeEach(() => {
+        // Get config
+        cy.apiGetConfig().then((response) => {
+            config = response.body;
+        });
```

Comment:
Please make this predetermined similar to referenced PR to easily [compare the before and after change](https://github.com/mattermost/mattermost-webapp/pull/4314/files#diff-e8e5668a9849acec0e1a98afb29a0883R48) 
```javascript
cy.apiUpdateConfig({
    TeamSettings: {
        SiteName: 'Mattermost',
    },
});
```